Regression: Accordion (from `@cds/angular`) lost `expanded` event declaration from version 5.5.2 to 5.5.3 (in d.ts file)

## Describe the bug

After updating Clarity from 5.5.1 to 5.6.2 I cannot longer use the `expanded` event of `<cds-accordion-panel>` because the `expandedChange` property is missing from the _d.ts_ file of `CdsAccordionPanelDirective`.

The build fails:

```html
  <cds-accordion-panel [expanded]="someState" (expandedChange)="setSomeState($event.detail)">...</cds-accordion-panel>
```

> error TS2339: Property 'detail' does not exist on type 'Event'.

## How to reproduce

Steps to reproduce the behavior:

1.  Go to https://stackblitz.com/edit/clarity-dark-theme-v5-a3szgg?devtoolsheight=33&file=src/app/app.component.html
2.  Wait for `npm install` / `ng serve` to finish.
3.  See build errors in terminal.

## Expected behavior

No build errors when using `expandedChange`'s `$event` with `<cds-accordion-panel>` like with versions up to 5.5.2.

Steps to see working state with version 5.5.2 in StackBlitz demo:

4.  Stop `ng serve` with ctrl+c.
5.  Change version of `@angular/cds` to 5.5.2 in _package.json_.
6.  Run `npm install` in terminal.
7.  Wait for `npm install` / `ng serve` to finish.
8.  Preview is usable and accordion works.

## Versions

**Clarity project:**

- [ ] Clarity Core
- [x] Clarity Angular/UI

**Clarity version:**

- [ ] v3.x
- [ ] v4.x
- [x] v5.x

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
Angular 11 and 12

--- a/packages/angular/projects/cds-angular/_stubs/directive.ts.mustache
+++ b/packages/angular/projects/cds-angular/_stubs/directive.ts.mustache
@@ -17,7 +17,7 @@ export class {{directiveClassName}} {
   }
   {{! empty string is added to support boolean attrs https://github.com/angular/angular/issues/14761 }}
   @Input() set {{name}}(value{{#isBoolean}}: boolean | ''{{/isBoolean}}) {
-    this.element.{{name}} = {{#isBoolean}}!!{{/isBoolean}}value;
+    this.element.{{name}} = {{#isBoolean}}value === '' ? true : {{/isBoolean}}value;
   };
   {{/props}}
 

--- a/packages/angular/projects/cds-angular/src/all.spec.ts
+++ b/packages/angular/projects/cds-angular/src/all.spec.ts
@@ -233,6 +233,19 @@ describe('CDS Button', () => {
   it('cds-button should be defined', () => {
     expect(instance.vcCdsButtonDirective).toBeDefined();
   });
+
+  it('boolean attributes should be treated as truthy', () => {
+    const element = fixture.nativeElement.querySelector('cds-button');
+    expect(instance.vcCdsButtonDirective.readonly).toBe(false);
+    expect(element.readonly).toBe(false);
+    expect(element.hasAttribute('readonly')).toBe(false);
+
+    element.setAttribute('readonly', '');
+    fixture.detectChanges();
+    expect(instance.vcCdsButtonDirective.readonly).toBe(true);
+    expect(element.readonly).toBe(true);
+    expect(element.hasAttribute('readonly')).toBe(true);
+  });
 });
 
 describe('CDS Icon Button', () => {

--- a/packages/core/src/accordion/accordion-panel.element.ts
+++ b/packages/core/src/accordion/accordion-panel.element.ts
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
@@ -71,7 +71,7 @@ export class CdsAccordionPanel extends LitElement {
 
   @querySlot('cds-accordion-content') private content: HTMLElement;
 
-  @event() protected expandedChange: EventEmitter<boolean>;
+  @event() expandedChange: EventEmitter<boolean>;
 
   private toggle() {
     this.expandedChange.emit(!this.expanded);

--- a/packages/core/src/alert/alert.element.ts
+++ b/packages/core/src/alert/alert.element.ts
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
@@ -133,7 +133,7 @@ export function getAlertContentLayout(
  * @cssprop --close-icon-color-hover
  */
 export class CdsAlert extends LitElement {
-  @event() private closeChange: EventEmitter<boolean>;
+  @event() closeChange: EventEmitter<boolean>;
 
   /**
    * Sets the overall height and width of the alert and icon based on value

--- a/packages/core/src/internal-components/overlay/overlay.element.ts
+++ b/packages/core/src/internal-components/overlay/overlay.element.ts
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
@@ -73,7 +73,7 @@ export class CdsInternalStaticOverlay extends CdsBaseFocusTrap {
 
   private overlayIdPrefix = '_overlay-';
 
-  @event() protected closeChange: EventEmitter<CloseChangeSources>;
+  @event() closeChange: EventEmitter<CloseChangeSources>;
 
   @state({ type: Boolean })
   protected isLayered = false;

--- a/scripts/core-ng-module-generator.js
+++ b/scripts/core-ng-module-generator.js
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2022 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
@@ -114,7 +114,7 @@ function createDirective(element, module) {
     directiveFileName: element.tagName,
     directiveClassName: `${element.name}Directive`,
     props: (element.members && getDirectiveProps(element.members)) || [],
-    events: (element.members && getDirectiveEvents(element.members)) || [],
+    events: (element.members && getDirectiveEvents(element.members, element.events)) || [],
     directiveFilePath: path.join(TARGET_PATH, path.dirname(module.path), `${element.tagName}.directive.ts`),
     directiveModule: path.dirname(module.path),
   };
@@ -133,11 +133,14 @@ function getDirectiveProps(props) {
     .map(prop => ({ name: prop.name, isBoolean: prop?.type?.text === 'boolean' }));
 }
 
-function getDirectiveEvents(events) {
-  return events
+function getDirectiveEvents(members = [], events = []) {
+  const memberEvents = members
     .filter(event => event.privacy === undefined) // public
     .filter(prop => prop.type && prop.type.text && prop.type.text.includes('EventEmitter')) // include only events
     .map(event => ({ name: event.name }));
+
+  // return distinct list of events when combining @output members and @event annotations
+  return Object.values([...memberEvents, ...events].reduce((prev, next) => ({ ...prev, [next.name]: next }), {}));
 }
 
 function writeToDisk(filename, content) {

